### ISSUE TITLE
Add an option to suppress the legal banner on each run

### ISSUE THREAD

**Issue by marektoman** [OmniSharp/omnisharp-vscode#3706]
## Steps to Reproduce ##
1. Open a C# project.
2. Run the project

## Expected Behavior ##
Seeing the uncluttered output of my code.

## Actual Behavior ##
```
-------------------------------------------------------------------
You may only use the Microsoft .NET Core Debugger (vsdbg) with
Visual Studio Code, Visual Studio or Visual Studio for Mac software
to help you develop and test your applications.
-------------------------------------------------------------------
```
```
my output
```

## Reason ##
The banner may provide useful information on the first 1-10 runs, but since then, at best it only consumes the precious space for your program output, at worst it reminds you that the first most important thing about whatever you are trying to achieve is the fact that MS employ a legal dept.

### OmniSharp log ###
<details>Post the output from Output-->OmniSharp log here</details>

### C# log ###
<details>Post the output from Output-->C# here</details>

## Environment information ##

**VSCode version**: 1.43.2
**C# Extension**: 1.21.16

<details><summary>Dotnet Information</summary>
.NET Core SDK (reflecting any global.json):
 Version:   3.1.201
 Commit:    b1768b4ae7

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18363
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.1.201\

Host (useful for support):
  Version: 3.1.3
  Commit:  4a9f85e9f8

.NET Core SDKs installed:
  2.1.509 [C:\Program Files\dotnet\sdk]
  3.1.101 [C:\Program Files\dotnet\sdk]
  3.1.200 [C:\Program Files\dotnet\sdk]
  3.1.201 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.1 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.1 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.1 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|csharp|ms-dotnettools|1.21.16|
|debugger-for-chrome|msjsdiag|4.12.6|
|dotnetscriptr|Wivuu|0.2.3|
|gc-excelviewer|GrapeCity|2.1.34|
|gitlens|eamodio|10.2.1|
|html-slim-scss-css-class-completion|gencer|1.5.23|
|lorem-ipsum|Tyriar|1.2.0|
|markdown-all-in-one|yzhang|2.7.0|
|markdown-pdf|yzane|1.4.4|
|monitor-file|rkostrzewski|1.0.1|
|mssql|ms-mssql|1.9.0|
|nunjucks-template|eseom|0.2.1|
|powershell|ms-vscode|2020.3.0|
|python|ms-python|2020.3.71659|
|quicktype|quicktype|12.0.46|
|remote-wsl|ms-vscode-remote|0.42.4|
|render-crlf|medo64|1.5.1|
|scss-refactoring|lukazakrajsek|0.1.4|
|sftp|liximomo|1.12.9|
|smarty|imperez|0.3.0|
|swig|zhutian|0.0.5|
|theme-sapphire|Tyriar|0.9.1|
|vs-code-xml-format|fabianlauer|0.1.5|
|vs-html-to-css|neptunedesign|1.0.0|
|vscode-auto-scroll|pejmannikram|1.0.3|
|vscode-autohotkey|slevesque|0.2.2|
|vscode-css-formatter|aeschli|1.0.1|
|vscode-docker|ms-azuretools|1.0.0|
|vscode-file-peek|abierbaum|1.0.1|
|vscode-great-icons|emmanuelbeziat|2.1.47|
|vscode-scss|mrmlnc|0.9.0|
|vscode-scss-formatter|sibiraj-s|2.0.0|
|vscodeintellicode|VisualStudioExptTeam|1.2.6|;

</details>

**Comment by cartermp** [comment]
Tagging for needing investigation - there is a legal matter to design an experience around here, since VSCode/C# doesn't require a EULA in the same way as VSWindows (where you don't see this banner).

**Comment by shivshanks** [comment]
I agree with the original poster and in fact I would argue that it should not show this after the very first time as it is running in VS Code which is a valid use case. I am even willing to go through some config (which I can set), that then triggers an EULA at next run to click and agree to make this go away.

The way it is right now, almost makes me want to stop using VS Code for C#.
